### PR TITLE
Ajacobs/remove pgsql

### PR DESCRIPTION
**Pull request by alecjacobs5401**
Removes the Kubernetes manifests for the `pgsql` and `codeintel-db` services. These are no longer necessary now that we have migrated `stage` to use RDS Postgres databases.

**TODOs Before Merging**:

- [ ] `kubectl delete` all the removed resources in the `stage:us-west-2:tooling-1` cluster in the `sourcegraph` namespace
- [ ] Ensure the PVCs are deleted (should happen above)

### CODE REVIEW

File: base/codeintel-db/README.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+## Important Notice
+
+Segment's self-hosted version of Sourcegraph uses an [External RDS Service](https://docs.sourcegraph.com/admin/external_services/postgres#kubernetes) for
+the `codeintel-db`.
+
+You can find the definition for that database in the [segmentio/terracode-tooling](https://github.com/segmentio/terracode-tooling) repository.
```
Comment (asaf-erlich) [inline base/codeintel-db/README.md]: In the two README links can you point to the specific file?

Comment (alecjacobs5401) [reply base/codeintel-db/README.md]: Sure, I thought about that and opted to just leave it generic but since you called it out I can 👍

Comment (alecjacobs5401) [reply base/codeintel-db/README.md]: Done in https://github.com/segmentio/deploy-sourcegraph/pull/14/commits/d90cd82cc0da165d014ff67a9e0165612ba0179d

File: base/codeintel-db/README.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+## Important Notice
+
+Segment's self-hosted version of Sourcegraph uses an [External RDS Service](https://docs.sourcegraph.com/admin/external_services/postgres#kubernetes) for
+the `codeintel-db`.
+
+You can find the definition for that database in the [segmentio/terracode-tooling](https://github.com/segmentio/terracode-tooling/blob/master/stage/us-west-2/sourcegraph/rds.tf) repository.
```
Comment (asaf-erlich) [inline base/codeintel-db/README.md]: Can you press 'y' on the github page so it points to a sha and not master? That way the link won't break if the file is ever moved or removed?

**Review by asaf-erlich — APPROVED** [review]
LGTM with the minor comment I wrote above about pointing to a sha and not master

File: base/codeintel-db/README.md (hunk 2 above)
Comment (alecjacobs5401) [reply base/codeintel-db/README.md]: TIL. Thats a super helpful keybind

**Comment by alecjacobs5401** [comment]
I removed the applied resources by checking out the `stage` branch and running 
```
kc delete --recursive -f base/pgsql -f base/codeintel-db
```
from the root of the repo

**Pull request merged**